web_widget_image_paste_clipboard/README.md

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
```

Comment:
being a new module, can you move this to new readme structure https://github.com/OCA/maintainer-tools/tree/master/template/module/readme ?